.github/workflows/release.yml

```diff
@@ -126,7 +126,7 @@ jobs:
         id: create_changelog
         with:
           config: cliff.toml
-          args: --bump
+          args: --bump --latest
```

Comment:
just FYI: I think this processes from the latest `release`, not `prerelease`. But that's what we want right?

Reply:
yes, I think that's what we want too. it worked fine locally at least.